vllm/platforms/rocm.py

```diff
@@ -102,6 +127,18 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
             else:
                 parallel_config.worker_cls = "vllm.worker.worker.Worker"
 
+    @classmethod
+    def verify_model_arch(cls, model_arch: str) -> None:
```

Comment:
Shouldn't it be `check_model_arch` here?

Reply:
(Though personally I prefer `verify_model_arch` as the method name.

Reply:
Thx! Done.